test/plugin.test.js

```diff
@@ -97,6 +98,61 @@ test('should log every query', async (t) => {
   })
 })
 
+test('should log batched queries', async (t) => {
+  const stream = split(JSON.parse)
+  stream.on('data', (line) => {
+    t.is(line.reqId, 'req-1')
+    switch (line.graphql.operationName) {
+      case 'QueryOne':
+        t.deepEqual(line.graphql, {
+          operationName: 'QueryOne',
+          queries: ['counter'],
+          variables: null
+        })
+        break
+      case 'QueryTwo':
+        t.deepEqual(line.graphql, {
+          operationName: 'QueryTwo',
+          queries: ['add'],
+          variables: { y: 2 }
+        })
+        break
+    }
+  })
+
+  const app = buildApp(t, { stream }, { logVariables: true })
+
+  const query1 = `query QueryOne {
+    counter
+  }`
+
+  const query2 = `query QueryTwo($y: Int!) {
+    four: add(x: 2, y: $y)
+  }`
+
+  const response = await app.inject({
+    method: 'POST',
+    headers: { 'content-type': 'application/json' },
+    url: '/graphql',
+    body: JSON.stringify([
+      { operationName: 'QueryOne', query: query1 },
+      { operationName: 'QueryTwo', query: query2, variables: { y: 2 } }
```

Comment:
Could we add a query3 item to test the error case too?

```suggestion
      { operationName: 'QueryTwo', query: query2, variables: { y: 2 } },
  {
    operationName: "BadQuery",
    query: "query DoubleQuery ($x: Int!) {---", // Malformed Query
  }
```